insight/pv_model/pv_model_class.py

```diff
@@ -20,23 +20,45 @@ def __init__(self, Iscn, Vocn, Imp, Vmp, Kv, Ki, Ns, Gn, G, Tn, T, Egap, err, in
         self.T = T
         self.Egap = Egap
         self.err = err
-        self.inverter = inverter
+        self.inverter_eff = inverter_eff
         # array with [number of strings, number in series (per string)]
         self.array_dim = array_dim
-
+        self.increment = increment
         self.q = constants.e
         self.k = constants.Boltzmann
 
     def calculate_currents(self, Rs, Rp, a, Iscn, Vocn, Ki, Kv, Vt, Ns, G, Gn, deltaT):
         """Calculates and returns the different current values of interest. Takes
         different PV module specifications and weather data as input.
+        
+        :param float Rs: The equivalent series resistance of the PV module.
+        :param float Rp: The equivalent parallel resistance of the PV module.
+        :param float a: The diode ideality constant.
+        :param float Iscn: The nominal short circuit current of the PV module.
+        :param float Vocn: The nominal open circuit voltage of the PV module.
+        :param float Ki: The rated current temperature coefficient.
+        :param flaot Kv: The rated voltage temperature coefficient.
+        :param float Vt: The thermal voltage of the PV module for Ns cells in series.
+        :param int Ns: The number of PV cells in series for the PV module.
+        :param float G: The irradiance of the environment surrounding the PV module.
+        :param float Gn: The nominal irradiance of the environment surrounding the PV module,
+            at 25 C degrees.
+        :param float deltaT: The difference between the nominal operating temperature and the
+            actual temeprature of the environment surrounding the PV module.
+        :return: Lists for the nominal and realistic output currents (Ipvn and Ipv, respectively),
+            the short circuit current impacted by environmental factors (Isc), and the 
+            resulting saturation current (I0).
         """
+        # so there seem to be two different ways of calculating Ipv and Isc, which one should we use? 
+        # we seem to use the first method everywhere else but then the new method only to calculate I0,
+            # does that make these values inconsistent with each other?
```

Comment:
Good questions! I'm not sure, but we can talk through it during tomorrow's meeting